src/main/kotlin/skytils/skytilsmod/features/impl/dungeons/DungeonFeatures.kt

```diff
@@ -324,9 +324,9 @@ class DungeonFeatures {
 
     @SubscribeEvent
     fun onSendChatMessage(event: SendChatMessageEvent) {
-        if (event.message.startsWith("/skytilscopyfail") && !event.addToChat) {
-            mc.thePlayer.addChatMessage(ChatComponentText("§aCopied selected death/fail to clipboard."))
-            GuiScreen.setClipboardString(event.message.substring("/skytilscopyfail ".length))
+        if (event.message.startsWith("/skytilscopy") && !event.addToChat) {
+            mc.thePlayer.addChatMessage(ChatComponentText("§9§lSkytils §8» §aCopied to clipboard."))
+            GuiScreen.setClipboardString(event.message.substring("/skytilscopy ".length))
```

Comment:
@My-Name-Is-Jeff Maybe we can eventually replace this with a subcommand within /skytils?

Reply:
The way it is right now, the user can't execute the command themselves, but we could do that